src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24414.cs

```diff
@@ -16,8 +16,34 @@ public Issue24414(TestDevice device)
 		[Category(UITestCategories.Visual)]
 		public void Issue24414Test()
```

Comment:
The test is failing in all the platforms by differences in the shadows:
![image](https://github.com/user-attachments/assets/b2281854-476e-4fc5-823b-488dfe8afe33)
Maybe commits https://github.com/dotnet/maui/pull/26789/commits/14eecfec039e25a519f83de7053b13fbb6a1d8dc after include the snapshots had an impact?

Could you review the tests? Let me know if needs help with it.

Reply:
Actually I have to capture all the shapshots because I changed the UI test.
I will on next commit+azp.